kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-lienzo/src/test/java/org/kie/workbench/common/stunner/client/lienzo/canvas/controls/LineSpliceAcceptorControlImplTest.java

```diff
@@ -0,0 +1,523 @@
+        final CompositeCommand<AbstractCanvasHandler, CanvasViolation> commands = commandsCapture.getValue();
+        final List<Command<AbstractCanvasHandler, CanvasViolation>> commandList = commands.getCommands();
+
+        assertEquals(commands.size(), 5);
+        assertEquals(commandList.stream().filter(cmd -> cmd instanceof CloneConnectorCommand).count(), 1);
+        assertEquals(commandList.stream().filter(cmd -> cmd instanceof DeleteControlPointCommand).count(), 0);
+        assertEquals(commandList.stream().filter(cmd -> cmd instanceof AddControlPointCommand).count(), 1);
+        assertEquals(commandList.stream().filter(cmd -> cmd instanceof SetConnectionTargetNodeCommand).count(), 1);
+        assertEquals(commandList.stream().filter(cmd -> cmd instanceof UpdateChildrenCommand).count(), 1);
+        assertEquals(commandList.stream().filter(cmd -> cmd instanceof UpdateElementPositionCommand).count(), 1);
+        assertEquals(parentNode, updateChildrenCommand.getParent());
+        assertEquals(spliceNode, updateChildrenCommand.getCandidates().iterator().next());
+    }
+
+    @Test
+    public void testAcceptTrueMultipleCPs() {
+        setCloneConnectorCommand();
+        setDeleteControlPointCommand();
+        setAddControlPointCommand();
+        setSetConnectionTargetNodeCommand();
+        setUpdateChildrenCommand();
+        setUpdateElementPositionCommand();
+        setExecuteTrue();
+
+        tested.init(canvasHandler);
+        Point2DArray firstHalfPoints = new Point2DArray();
+        Point2DArray secondHalfPoints = new Point2DArray();
+        Point2D CP0 = new Point2D(10, 10);
+        Point2D CP1 = new Point2D(20, 10);
+        Point2D CP2 = new Point2D(30, 10);
+        Point2D CP3 = new Point2D(40, 10);
+        Point2D CP4 = new Point2D(50, 10);
+        Point2D CP5 = new Point2D(60, 10);
+        firstHalfPoints.push(CP0);
+        firstHalfPoints.push(CP1);
+        firstHalfPoints.push(CP2);
+        firstHalfPoints.push(CP3);
+        secondHalfPoints.push(CP4);
+        secondHalfPoints.push(CP5);
+
+        assertTrue(tested.accept(spliceNode,
+                                 location,
+                                 parentNode,
+                                 connector,
+                                 6, // Points between the first and last are deleted
+                                 firstHalfPoints,
+                                 secondHalfPoints));
+
+        verify(highlight, times(2)).unhighLight();
+        verify(highlight, times(1)).highLight(connector);
+        verify(commandManager, times(1))
+                .execute(eq(canvasHandler), any(CompositeCommand.class));
+
+        verify(tested).executeCommands(any(Function.class), commandsCapture.capture());
+        final CompositeCommand<AbstractCanvasHandler, CanvasViolation> commands = commandsCapture.getValue();
+        final List<Command<AbstractCanvasHandler, CanvasViolation>> commandList = commands.getCommands();
+
```

Comment:
Hey @domhanak,
This is the point I'm checking if all the commands needed were properly added to the batch to perform the reconnetions and the containment. This test covers the most complete scenatio. 
The idea is to ensure the all the commands for each scenatio are there.
Thanks